### ISSUE TITLE
[BUG] Failing CI Test

### ISSUE THREAD

**Issue by lunarcloud** [lunarcloud/trek-character-icon-creator#53]
## Bug Description
Accessibility tests sometimes fail.

This is alarming and we should try to avoid it.

## Example 
https://github.com/lunarcloud/trek-character-icon-creator/actions/runs/22209664734/job/64241016449?pr=52 

36s
Run npm test
> test
> playwright test
[WebServer] node:internal/errors:496
[WebServer]     ErrorCaptureStackTrace(err);
[WebServer]     ^
[WebServer] 
[WebServer] TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
[WebServer]     at new NodeError (node:internal/errors:405:5)
[WebServer]     at watch (node:fs:2414:11)
[WebServer]     at file:///home/runner/work/trek-character-icon-creator/trek-character-icon-creator/src/watch-html.js:33:1
[WebServer]     at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
[WebServer]     at async ModuleLoader.import (node:internal/modules/esm/loader:337:24)
[WebServer]     at async loadESM (node:internal/process/esm_loader:34:7)
[WebServer]     at async handleMainPromise (node:internal/modules/run_main:106:12) {
[WebServer]   code: 'ERR_FEATURE_UNAVAILABLE_ON_PLATFORM'
[WebServer] }
[WebServer] 
[WebServer] Node.js v18.20.8
Running 62 tests using 1 worker
